subprojects/gdk_rust/gdk_electrum/src/account.rs

```diff
@@ -773,39 +789,72 @@ pub fn create_tx(
         }
     }
 
-    if request.addressees.is_empty() {
-        return Err(Error::EmptyAddressees);
-    }
-
-    if request.previous_transaction.is_some() {
-        return Err(Error::Generic("bump not supported".into()));
-    }
-
     let send_all = request.send_all.unwrap_or(false);
     request.send_all = Some(send_all); // accept default false, but always return the value
-    if !send_all && request.addressees.iter().any(|a| a.satoshi == 0) {
-        return Err(Error::InvalidAmount);
-    }
 
-    if !send_all {
-        for address_amount in request.addressees.iter() {
-            if address_amount.satoshi <= DUST_VALUE {
-                match network.id() {
-                    NetworkId::Bitcoin(_) => return Err(Error::InvalidAmount),
-                    NetworkId::Elements(_) => {
-                        if address_amount.asset_id == network.policy_asset {
-                            // we apply dust rules for liquid bitcoin as elements do
-                            return Err(Error::InvalidAmount);
+    let mut template_tx = None;
+    let mut change_addresses = HashMap::new();
+
+    // When a previous transaction is replaced, use it as a template for the new transaction
+    if let Some(ref prev_txitem) = request.previous_transaction {
+        if !request.addressees.is_empty() || send_all {
+            return Err(Error::InvalidReplacementRequest);
+        }
+
+        let prev_tx = BETransaction::from_hex(&prev_txitem.transaction, network.id())?;
+        let policy_asset = network.policy_asset.as_deref().unwrap_or("btc");
```

Comment:
Elements/Liquid rbf support will not be exposed.
Do you prefer to leave the code to handle that anyway?

Reply:
I kept the code to handle it, but made it explicitly reject tx replacement for liquid networks in https://github.com/Blockstream/gdk/pull/119/commits/56db1ed00cd7ca8519a8dbb04ca7cb6a474330ed.